packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -109,6 +111,13 @@ export class StatusBarImpl extends ReactWidget implements StatusBar {
         this.node.style.backgroundColor = this.backgroundColor ? this.backgroundColor : null;
     }
 
+    async setColor(color?: string): Promise<void> {
```

Comment:
`setBackgroundColor` is waiting for `this.ready`, should `setColor` do the same? As well as defining an internal method just to mimic what is done in `setBackgroundColor`?

Reply:
thanks I must have missed it in my updates